components/CaseStatus/AddCaseStatusForm/AddCaseStatusForm.tsx

```diff
@@ -0,0 +1,82 @@
+import { Form, Formik, FormikHelpers, FormikValues } from 'formik';
+import CASE_STATUS from 'data/flexibleForms/caseStatus';
+import { generateInitialValues } from 'lib/utils';
+import { generateFlexibleSchema } from 'lib/validators';
+import FlexibleField from 'components/FlexibleForms/FlexibleFields';
+import Button from 'components/Button/Button';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+const AddCaseStatusForm: React.FC<{
+  personId: number;
+  prefilledField: any;
```

Comment:
Minor: What do you think about `prefilledFields` instead? So it's clear it's multiple values.

Reply:
Changed to plural!